### PR TITLE
add my redirects for new config pages

### PR DESCRIPTION
**Pull request by zsarnett**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/my/ha-panel-my.ts [hunk 1]
```diff
@@ -129,17 +132,35 @@ export const getMyRedirects = (hasSupervisor: boolean): Redirects => ({
...
+  },
+  analytics: {
+    redirect: "/config/analytics",
+  },
+  updates: {
+    redirect: "/config/updates",
+  },
+  system_dashboard: {
+    redirect: "/config/system",
+  },
```
Comment (balloob) [inline src/panels/my/ha-panel-my.ts:163]: don't forget to add them to https://github.com/home-assistant/my.home-assistant.io/blob/main/redirect.json

Comment (balloob) [reply src/panels/my/ha-panel-my.ts:163]: And we should update https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/analytics.markdown to point at the new analytics My link.

Comment (ludeeus) [reply src/panels/my/ha-panel-my.ts:163]: <https://github.com/home-assistant/my.home-assistant.io/blob/main/redirect.json> we cant yet as it does not exist

Comment (balloob) [reply src/panels/my/ha-panel-my.ts:163]: Right, but we should get a draft PR ready.

Comment (ludeeus) [reply src/panels/my/ha-panel-my.ts:163]: https://github.com/home-assistant/home-assistant.io/pull/22553

**Review by bramkragten — APPROVED** [review]

File: src/panels/my/ha-panel-my.ts (hunk 1 above)
Comment (ludeeus) [reply src/panels/my/ha-panel-my.ts:163]: https://github.com/home-assistant/my.home-assistant.io/pull/214

**Pull request merged**